lib/PrincipleStudios.OpenApi.CSharp/Templates/enumModel.handlebars

```diff
@@ -9,7 +9,9 @@ namespace {{this.PackageName}}
     /// {{linewrap Description "
     /// "}}
     /// </summary>{{/if}}
-    {{#if Model.IsString}}[global::System.Text.Json.Serialization.JsonConverter(typeof(global::PrincipleStudios.OpenApiCodegen.Json.Extensions.JsonStringEnumPropertyNameConverter))]{{/if}}
+    {{#if Model.IsString}}[global::System.Text.Json.Serialization.JsonConverter(typeof(global::PrincipleStudios.OpenApiCodegen.Json.Extensions.JsonStringEnumPropertyNameConverter))]
+    [global::System.ComponentModel.TypeConverter("{{../this.PackageName}}.{{ClassName}}TypeConverter")]
```

Comment:
Interestingly, the `TypeConverterAttribute` used here can also use `typeof`, but that didn't seem to work, and I'm not sure why. Something about assembly loading? I'm guessing since the unit tests are a dynamic compilation in-memory, it causes that to do something weird. I need to learn how `typeof` works in attributes more, apparently.

Reply:
It's apparently something specific with `TypeConverterAttribute`.

[`TypeConverterAttribute(Type)`](https://github.com/microsoft/referencesource/blob/master/System/compmod/system/componentmodel/TypeConverterAttribute.cs#L51-L53)
[`JsonConverterAttribute(Type)`](https://github.com/dotnet/runtime/blob/main/src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Attributes/JsonConverterAttribute.cs#L25-L28)